src/Blorc.OpenIdConnect/wwwroot/oidc-client-interop.js

```diff
@@ -25,10 +25,35 @@
                     config.silent_redirect_uri += "/_content/Blorc.OpenIdConnect/silent-refresh.html";
                 }
 
+                let self = this;
+
+                if (config.timeForAutomaticLogoutOnUserInactivity > 0) {
+                    var inactivityTimer = setTimeout(trySignoutRedirect, config.timeForAutomaticLogoutOnUserInactivity);
+
+                    document.onmousemove = resetInactivityTimer;
```

Comment:
can this be overwritten by something? Is there a different way to subscribe (just once)?

Reply:
Fixed this addListener